pkgs/development/compilers/gcc/builder.sh

```diff
@@ -285,7 +285,8 @@ postInstall() {
     # Disable RANDMMAP on grsec, which causes segfaults when using
     # precompiled headers.
     # See https://bugs.gentoo.org/show_bug.cgi?id=301299#c31
-    paxmark r $out/libexec/gcc/*/*/{cc1,cc1plus}
+    # Error is ignored because paxmark fails on 'mips64el' with "not a valid ELF executable (invalid PT_ entry:7)"
```

Comment:
If paxmark is broken on mips64 then it should be globally disabled just like we disable paxmark globally on darwin.